### ISSUE TITLE
Seems installing the plugin causes http://www.redmine.org/issues/4942 bug 

### ISSUE THREAD

**Issue by aerojustaero** [mully/redmine_harvest#2]

**Comment by squeejee** [comment]
Thanks for bringing this to my attention.  Definitely an issue.  Looking into it.

**Comment by newism** [comment]
To confirm I'm also having this issue with the 0.9.2

**Comment by newism** [comment]
Did you ever have a chance to look at this issue? A fix would be stupendous!

**Comment by squeejee** [comment]
No, I never did.  I need to upgrade to the 1.x release of Redmine and get it working with that.  Hopefully, I will get to it sooner rather than later.  It is a very annoying bug.

**Comment by leevigraham** [comment]
Oh man you've made my day!

**Comment by szajbus** [comment]
The bug is caused by Mash gem that reimplements Hash#stringify_keys! method. Mash is internally used by Harvestr gem, which this plugin relies on.

I have rewritten parts of this plugin so it now uses Harvest gem instead of Harvestr. Check out the commit.
https://github.com/szajbus/redmine_harvest/commit/366880ab7a8f2981f647accbbbc09d26218d15d7

**Comment by leevigraham** [comment]
Thanks for the fix! Unfortunately we've moved on from Redmine to Jira + Greenhopper. For $20 a year and all profits going to charity it was hard to pass up.

**Comment by squeejee** [comment]
Excellent news!  

Thanks for taking the initiative and fixing this!
